README.md

```diff
@@ -893,6 +895,8 @@ Buffer-local mappings also take a buffer number as their first argument (`0` set
 ```lua
 vim.api.nvim_set_keymap('n', '<leader><Space>', ':set hlsearch!<CR>', { noremap = true, silent = true })
 -- :nnoremap <silent> <leader><Space> :set hlsearch<CR>
+vim.api.nvim_set_keymap('n', '<leader>tegf',  [[<Cmd>lua require('telescope.builtin').git_files()<CR>]], { noremap = true, silent = true })
+-- :nnoremap <silent> <leader>tegf :lua require('telescope.builtin').git_files()<CR>
```

Comment:
```suggestion
-- :nnoremap <silent> <leader>tegf <Cmd>lua require('telescope.builtin').git_files()<CR>
```

Reply:
fixed in https://github.com/nanotee/nvim-lua-guide/commit/39c67605f6361e61ca4158ec46ee144f4f356c97